src/Hedwig/ClientApp/src/components/ProcessList/_index.scss

```
@@ -0,0 +1,57 @@
+// Derived from VA.gov Design System: https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/master/packages/formation/sass/modules/_m-process-list.scss
+
+.process {
+	list-style: none;
+	padding: 1em 0;
+	position: relative;
+
+	// Exception for when the paragraph is then
+	// lone child. Mostly found in Playbook
+	p {
+		padding: 0;
+		margin-top: 0;
+
+		&:only-of-type {
+			margin-top: 0;
+		}
+	}
+
+	ul {
+		margin: 0 0 1em 1.25em !important;
+		padding: 0;
+
+		li {
+			list-style: square;
+			margin: 0;
+			padding: 0.1em 0;
+		}
+	}
+}
+
+.process-step {
+	border-left: 0.2em solid $theme-color-primary;
```

Part of the issue was that we were ending up with a 3.2px border. This will give us 3px, assuming a 16px font-size, which will be slightly easier to center on.

```suggestion
	border-left: 0.1875em solid $theme-color-primary;
```